### ISSUE TITLE
Several log softmax failures in benchdnn

### ISSUE THREAD

**Issue by annop-w** [oneapi-src/oneDNN#1545]
# Summary
Following the change in [`b44df87eb`](https://github.com/oneapi-src/oneDNN/commit/b44df87ebed2be20b43000464c72f8f5d998f0bf) several cases of log softmax fail benchdnn due to comparison of `-inf`. 

# Version
```
onednn_verbose,info,oneDNN v3.0.0 (commit b44df87ebed2be20b43000464c72f8f5d998f0bf)
onednn_verbose,info,cpu,runtime:OpenMP,nthr:16
onednn_verbose,info,cpu,isa:AArch64 SVE (256 bits)
```

# Environment
* CPU
```
Architecture:                    aarch64
CPU op-mode(s):                  32-bit, 64-bit
Byte Order:                      Little Endian
CPU(s):                          16
On-line CPU(s) list:             0-15
Thread(s) per core:              1
Core(s) per socket:              16
Socket(s):                       1
NUMA node(s):                    1
Vendor ID:                       ARM
Model:                           1
Stepping:                        r1p1
BogoMIPS:                        2100.00
L1d cache:                       1 MiB
L1i cache:                       1 MiB
L2 cache:                        16 MiB
L3 cache:                        32 MiB
NUMA node0 CPU(s):               0-15
Vulnerability Itlb multihit:     Not affected
Vulnerability L1tf:              Not affected
Vulnerability Mds:               Not affected
Vulnerability Meltdown:          Not affected
Vulnerability Mmio stale data:   Not affected
Vulnerability Retbleed:          Not affected
Vulnerability Spec store bypass: Mitigation; Speculative Store Bypass disabled via prctl
Vulnerability Spectre v1:        Mitigation; __user pointer sanitization
Vulnerability Spectre v2:        Mitigation; CSV2, BHB
Vulnerability Srbds:             Not affected
Vulnerability Tsx async abort:   Not affected
Flags:                           fp asimd evtstrm aes pmull sha1 sha2 crc32 atomics fphp asimdhp cpuid asimdrdm jscvt fcma lrcpc dcpop sha3 sm3 sm4 asimddp sha512 sve asimdfhm dit uscat ilrcpc flagm ssbs paca
                                 pacg dcpodp svei8mm svebf16 i8mm bf16 dgh rng
```
* OS version
```
Linux ip-10-252-30-171 5.15.0-1028-aws #32~20.04.1-Ubuntu SMP Mon Jan 9 18:02:12 UTC 2023 aarch64 aarch64 aarch64 GNU/Linux
```
* Compiler version
```
gcc-10 (Ubuntu 10.3.0-1ubuntu1~20.04) 10.3.0
Copyright (C) 2020 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
```
* CMake version
```
cmake version 3.16.3
```
* git hash
```
b44df87ebed2be20b43000464c72f8f5d998f0bf
```

# Steps to reproduce
```
tests/benchdnn/benchdnn --softmax --sdt=f16 --alg=LOGSOFTMAX --axis=0 16x16
```

# Observed behavior
```
onednn_verbose,info,oneDNN v3.0.0 (commit b44df87ebed2be20b43000464c72f8f5d998f0bf)
onednn_verbose,info,cpu,runtime:OpenMP,nthr:16
onednn_verbose,info,cpu,isa:AArch64 SVE (256 bits)
onednn_verbose,info,gpu,runtime:none
onednn_verbose,info,prim_template:operation,engine,primitive,implementation,prop_kind,memory_descriptors,attributes,auxiliary,problem_desc,exec_time
onednn_verbose,exec,cpu,reorder,simple:any,undef,src_f32::blocked:ab:f0 dst_f16::blocked:ab:f0,,,16x16,0.0581055
onednn_verbose,exec,cpu,softmax,ref:any,forward_training,src_f16::blocked:ab:f0 dst_f32::blocked:ab:f0,,alg:softmax_log axis:0,16x16,0.176025
onednn_verbose,exec,cpu,reorder,simple:any,undef,src_f32::blocked:ab:f0 dst_f32::blocked:ab:f0,,,16x16,0.180176
[   0][DST][0:0] exp_f32:        -inf exp:        -inf got:        -inf diff:     nan rdiff:     nan
[   1][DST][0:1] exp_f32:        -inf exp:        -inf got:        -inf diff:     nan rdiff:     nan
[   2][DST][0:2] exp_f32:        -inf exp:        -inf got:        -inf diff:     nan rdiff:     nan
[   3][DST][0:3] exp_f32:        -inf exp:        -inf got:        -inf diff:     nan rdiff:     nan
[   4][DST][0:4] exp_f32:        -inf exp:        -inf got:        -inf diff:     nan rdiff:     nan
[   5][DST][0:5] exp_f32:        -inf exp:        -inf got:        -inf diff:     nan rdiff:     nan
[   6][DST][0:6] exp_f32:        -inf exp:        -inf got:        -inf diff:     nan rdiff:     nan
[   8][DST][0:8] exp_f32:        -inf exp:        -inf got:        -inf diff:     nan rdiff:     nan
[   9][DST][0:9] exp_f32:        -inf exp:        -inf got:        -inf diff:     nan rdiff:     nan
0:FAILED (errors:73 total:256) __REPRO: --softmax --sdt=f16 --alg=LOGSOFTMAX --axis=0 16x16
tests:1 passed:0 skipped:0 mistrusted:0 unimplemented:0 invalid_arguments:0 failed:1 listed:0
total compute_ref: sum(s):0.00
```

# Expected behavior
```
0:PASSED __REPRO: --softmax --sdt=f16 --alg=LOGSOFTMAX --axis=0 16x16
tests:1 passed:1 skipped:0 mistrusted:0 unimplemented:0 invalid_arguments:0 failed:0 listed:0
total compute_ref: sum(s):0.00
```

**Comment by vpirogov** [comment]
These fails should be addressed by 047f457a9caba052b604f3a5533fe07cb22cdbdd.

**Comment by annop-w** [comment]
This has been fixed. Thank you.
